README.adoc

```diff
@@ -1,14 +1,41 @@
-# microprofile-telemetry
+//
+// Copyright (c) 2022-2022 Contributors to the Eclipse Foundation
```

Comment:
```suggestion
// Copyright (c) 2022 Contributors to the Eclipse Foundation
```
When we change this again, we will then add the beginning year and end year in the format of beginning-end year

Reply:
Ok, I will change that too.